templates/packer/ansible/roles/common/files/worker-initialization-scripts/worker_builder.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euo pipefail
+source /tmp/cloud_init_vars
+
+RUN_BUILDER=${RUN_BUILDER:-}
+if [[ -z "$RUN_BUILDER" ]] || [ "$RUN_BUILDER" != true ]; then
+  echo "RUN_BUILDER not defined or not set to 'true', not starting builder."
+  exit 0
+fi
+
+echo "Starting osbuild-jobsite-builder."
+/usr/libexec/osbuild-composer/osbuild-jobsite-builder &
+disown $!
```

Comment:
What about running this in its own service? Enabled in the playbook, but running only when `ConditionPathExists` is true (the fleet would create a file, something like `/tmp/run_builder_service`).